scripts/fast-reboot

```diff
@@ -533,9 +528,11 @@ sonic_asic_type=$(sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)
 BOOT_TYPE_ARG="cold"
 case "$REBOOT_TYPE" in
     "fast-reboot")
+        check_warm_restart_in_progress
```

Comment:
do we need check if fastboot is already in progress?

Reply:
There is a timeframe limitation to running fast-reboot - it will fail if triggered within this time.